viz.Rmd

```diff
@@ -55,7 +64,7 @@ As with most coding tasks, it is totally fine (and quite common) to make mistake
 
 - **line plots** visualize trends with respect to an independent, ordered quantity (e.g. time)
```

Comment:
I think we should make this list align with the order that we introduce them in the chapter
Also should we group the graphs? E.g. put all the scatterplots together etc (co2 data + eruption + canlang)

Reply:
Addressed the order. I do think we could move `canlang` to being after co2 data. But eruption is not at all about a scatter plot, that is about saving data. So we should leave that one at the end.

Reply:
Duh, I am silly, the eruption data is there too. Yes, I agree, let's move islands to after canlang!